services/persistence/main.py

```diff
@@ -42,10 +63,36 @@ def texts_and_labels():
     return jsonify({'n_inserted': n_inserts}), 201
 
 
-@app.route("/texts", methods=['GET'])
-def texts():
-    n_texts = request.args.get('n')
-    text_data = {'data': get_texts(n_texts)}
+@app.route("/prioritized_texts", methods=['GET'])
+def prioritized_texts():
+    """
+    Retrieves all of the political statements from the database, sends them to the manifesto model to be prioritized
+    and returns at most as many texts given in the GET parameter `n`, ordered by their uncertainty.
+
+    :return: {
+        'data': [
+            {'text_id': 1, 'label': 'left'},
+            ...
+        ]
+    }
+    """
+    n_texts = int(request.args.get('n'))
+    texts = get_texts_only(1000000)
+
+    samples = {
+        'data': list(map(lambda entry: {'text_id': entry['text_id'], 'text': entry['statement']}, texts))
+    }
+
+    # ask model about sample uncertainty
+    url = 'http://manifesto_model:{}/estimate_uncertainty'.format(MANIFESTO_MODEL_HTTP_PORT)
```

Comment:
wenn die vorhersage sich eh nich aendert, bis das model neu trainiert wird, koennte man die prioritization auch direkt beim model training machen - vorausgesetzt, es kommen keine neuen daten zwischendurch oder?

Reply:
Ja können wir so machen, dann würde ich bis die Richtung umdrehen, d.h. die Persistence triggert das Training und nicht umgekehrt.
Neue Daten, also neue Texte?

Reply:
ja meinte neue texte. 

und waer ok, wenn persistence das triggert, oder? oder spaeter vielleicht der mechanismus, der neue daten holt. aber all X-minuten, so wie's jetzt is, is ja auch perfekt. 

meine sorge war nur, dass irgendwann die session zu lange zu laden dauert.

Reply:
die sessiondauer, also das initiale laden der daten in der ui ist im moment noch akzeptabel. Koennten dafuer noch gzip compression anmachen, laeuft ja im moment alles auf einem host. oder halt umdrehen.

Ich hatte gestern die erste variante so, dass die persistence das training anstoesst, dafuer brauchten wir dann aber zusaetzlichen zustand (also eine queue der neu gelabelten beispiele). an fuer sich ist das netter, weil das uns erlaubt, ein online training zu implementieren, also das modell nicht jedes mal komplett fitten.